test/java/io/envoyproxy/envoymobile/engine/testing/QuicTestServerTest.java

```diff
@@ -111,7 +111,7 @@ public void setUpEngine() throws Exception {
     QuicTestServer.startQuicTestServer();
     CountDownLatch latch = new CountDownLatch(1);
     engine = new AndroidEngineBuilder(appContext, new Custom(config))
-                 .addLogLevel(LogLevel.TRACE)
+                 .addLogLevel(LogLevel.OFF)
```

Comment:
why did you turn the log off?

Reply:
There is a limit for logs on CI. The deal is to turn them on when debugging.

Reply:
Oh ok. But what if there's a production crash? Wouldn't we need this log? Or there's a different mechanism for logging in prod?

Reply:
This is only for tests.

Reply:
Oh true, I overlooked that, lol. Ok then, thank you!.